jupyter-archimedes/src/dune

```diff
@@ -5,6 +5,7 @@
  (modes       byte)
  (modules     Jupyter_archimedes)
  (flags       ((:include %{workspace_root}/config/ocaml_flags.sexp)))
+ (optional)
```

Comment:
Why this? The opam build does `"-p" name`, in that way it already prevents building `jupyter-archimedes` when building `jupyter`. Optional should be used when some parts of yur library are to be built only when all its dependencies a present.

Reply:
I have the same question. Why is `(optional)` required?

Reply:
You can build the project without having archimedes installed.

Reply:
But you can already do that with `dune build -p jupyter`

Reply:
Yes. I can remove it if you want. I have no special opinion actually. I added I found it nice since it's an optional part of the library but no special feelings about it :)

Reply:
Please remove it. I didn't think it is needed.